eclipse-theia/theia
# monaco: fix symbol icons

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/11357.

The pull-request fixes the symbol icons for multiple views (outline, callhierarchy, treehierarchy) by re-using codicons similarly to vscode. Previously we used `svg` which were removed from our sources so we defaulted to an incorrect icon.

_outline_:

![outline-fix](https://user-images.githubusercontent.com/40359487/176257969-8310d185-0e5b-4a62-8efa-f2eda1ab8dfb.png)

_call-hierarchy_:

![call-hierarchy-fix](https://user-images.githubusercontent.com/40359487/176257989-856226b5-f033-45b4-ba96-47255bae137d.png)

#### How to test


1. start the application
2. open multiple files, for each file opened confirm that the symbols in the outline-view for the file are correct
3. confirm that expanding/collapsing nodes in the outline view correctly display their symbols
4. confirm that the `call-hierarchy` symbols are correct

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
